sdks/java/extensions/python/src/main/java/org/apache/beam/sdk/extensions/python/PythonService.java

```diff
@@ -132,6 +136,17 @@ private String whichPython() {
     throw new RuntimeException("Unable to find a suitable Python executable.");
   }
 
+  @VisibleForTesting
+  static String getMatchingStablePythonSDKVersion(String javaSDKVersion) {
+    if (javaSDKVersion == null) {
```

Comment:
I'm curious when this can happen.

Reply:
It's just a safeguard for the case that `sdk_version` field is missing from `sdk.properties` file. In practice, this should not happen.